### DISCUSSION TITLE
Podman support

### DISCUSSION THREAD

**Discussion by ghost** [dragonflydb/dragonfly#2064] [Ideas]
https://podman.io is a rootless container solution, would be interesting to support it as well.
Should be drop-in replacement for Docker.

![image](https://github.com/dragonflydb/dragonfly/assets/74614193/df21af32-e37b-44b0-9952-2bfde138534f)

**Comment by romange** [comment]
if it's a drop in replacement, why not using it with dragonfly image? Have you tried and experienced problems?
